.gitignore

```diff
@@ -11,6 +11,7 @@ workflow_run_jobs
 # Python bytecode.
 *.pyc
 __pycache__
+.idea
```

Comment:
IDEA is not Python bytecode. Please put it under a separate section.

```
# IDEA
.idea
```

Reply:
Thank you! I fixed it